riscv/encoding.h

```diff
@@ -1844,6 +1844,673 @@
 #define MASK_VL4R_V  0xfff0707f
 #define MATCH_VL8R_V 0x1e807007
 #define MASK_VL8R_V  0xfff0707f
+#define MATCH_ADD8 0x4800007f
+#define MASK_ADD8 0xfe00707f
+#define MATCH_ADD16 0x4000007f
+#define MASK_ADD16 0xfe00707f
+#define MATCH_ADD64 0xc000107f
```

Comment:
It seems that some opcode belong to zp64 subset and some are included in rv64 profile.  However, all of them don't have implementation.   If there are some instructions have only opcode, please remove them from encoding.h and send them back with real implementation.

Reply:
OK. I will remove the opcode that is not implemented now.